### ISSUE TITLE
Do not require variable values for "sls help" or other no-op functions

### ISSUE THREAD

**Issue by chris-hailstorm** [serverless/serverless#2406]
Request command line help:

```
$ sls help
 
  Serverless Error ---------------------------------------
 
     Trying to populate non string value into a string for
     variable ${opt:stage}. Please make sure the value of
     the property is a string.
 
  Get Support --------------------------------------------
     Docs:          docs.serverless.com
     Bugs:          github.com/serverless/serverless/issues
```

This is apparently because my serverless.yml file includes `${opt:stage}` variables references.  If I try again with: `sls help --stage=xyz` help works as expected.

There should be no need to resolve variables references when the "help" command doesn't need the variables.  The same may be true of other sls commands, I'm not sure about that.

**Comment by rowanu** [comment]
What SLS version ware you running?

This command works fine for me in the latest version, as it was dealt with in #1721.

Please follow the issue template in the future.

**Comment by chris-hailstorm** [comment]
Issue resolved in 1.0.2
